cypress/support/clean.commands.js

```diff
@@ -20,19 +47,53 @@ Cypress.Commands.add('cleanCountries', (url) => {
   })
 })
 
-Cypress.Commands.add('cleanBiography', (url) => {
+Cypress.Commands.add('cleanNames', (url) => {
   cy.getCookie('XSRF-TOKEN').then((cookie) => {
     cy.request({
       method: 'POST',
-      url: '/account/biographyForm.json', // baseUrl is prepended to url
+      url: '/account/nameForm.json', // baseUrl is prepended to url
       body: {
         errors: [],
-        biography: {
+        givenNames: {
           errors: [],
           getRequiredMessage: null,
           required: true,
           value: '',
         },
+        familyName: {
+          errors: [],
+          getRequiredMessage: null,
+          required: true,
+          value: '',
+        },
+        creditName: {
+          errors: [],
+          getRequiredMessage: null,
+          required: true,
+          value: '',
+        },
+        visibility: {
+          errors: [],
+          required: true,
+          getRequiredMessage: null,
+          visibility: 'LIMITED',
+        },
+      },
+      headers: {
+        'X-XSRF-TOKEN': cookie.value,
+      },
+    })
+  })
+})
+
+Cypress.Commands.add('cleanOtherNames', (url) => {
```

Comment:
We are not using the URL parameter.
(I am deleting the other ones on my last PR)